src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonic.hpp

```diff
@@ -0,0 +1,295 @@
+#include "Time/Actions/ChangeStepSize.hpp"         // IWYU pragma: keep
+#include "Time/Actions/RecordTimeStepperData.hpp"  // IWYU pragma: keep
+#include "Time/Actions/SelfStartActions.hpp"       // IWYU pragma: keep
+#include "Time/Actions/UpdateU.hpp"                // IWYU pragma: keep
+#include "Time/StepChoosers/Cfl.hpp"               // IWYU pragma: keep
+#include "Time/StepChoosers/Constant.hpp"          // IWYU pragma: keep
+#include "Time/StepChoosers/Increase.hpp"          // IWYU pragma: keep
+#include "Time/StepChoosers/StepChooser.hpp"
+#include "Time/StepControllers/StepController.hpp"
+#include "Time/Tags.hpp"
+#include "Time/TimeSteppers/TimeStepper.hpp"
+#include "Time/Triggers/TimeTriggers.hpp"
+#include "Utilities/Functional.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace Frame {
+// IWYU pragma: no_forward_declare MathFunction
+struct Inertial;
+}  // namespace Frame
+namespace Parallel {
+template <typename Metavariables>
+class CProxy_ConstGlobalCache;
+}  // namespace Parallel
+/// \endcond
+
+struct EvolutionMetavars {
+  static constexpr int volume_dim = 3;
+  using frame = Frame::Inertial;
+  using system = GeneralizedHarmonic::System<volume_dim>;
+  using temporal_id = Tags::TimeStepId;
+  static constexpr bool local_time_stepping = false;
+  using initial_data_tag = Tags::AnalyticSolution<
+      GeneralizedHarmonic::Solutions::WrappedGr<gr::Solutions::KerrSchild>>;
+  using boundary_condition_tag = initial_data_tag;
+  using normal_dot_numerical_flux =
+      Tags::NumericalFlux<GeneralizedHarmonic::UpwindFlux<volume_dim>>;
+  using events = tmpl::list<
+      dg::Events::Registrars::ObserveErrorNorms<
+          volume_dim, typename system::variables_tag::tags_list>,
+      dg::Events::Registrars::ObserveFields<
+          volume_dim,
+          tmpl::append<
+              typename system::variables_tag::tags_list,
+              tmpl::list<::Tags::PointwiseL2Norm<
+                             GeneralizedHarmonic::Tags::GaugeConstraint<
+                                 volume_dim, frame>>,
+                         ::Tags::PointwiseL2Norm<
+                             GeneralizedHarmonic::Tags::ThreeIndexConstraint<
+                                 volume_dim, frame>>,
+                         ::Tags::PointwiseL2Norm<
+                             GeneralizedHarmonic::Tags::FourIndexConstraint<
+                                 volume_dim, frame>>>>,
+          typename system::variables_tag::tags_list>>;
+  using triggers = Triggers::time_triggers;
+
+  // A tmpl::list of tags to be added to the ConstGlobalCache by the
+  // metavariables
+  using const_global_cache_tags = tmpl::list<
+      initial_data_tag, Tags::TimeStepper<TimeStepper>,
+      // When using local timestepping, replace the previous tag with the
+      // following two lines.
+      // Tags::TimeStepper<tmpl::conditional_t<local_time_stepping,
+      //                                       LtsTimeStepper, TimeStepper>>,
```

Comment:
I asked this in another comment before, have you actually tested local time stepping with this executable? If no, please just remove this code.

Reply:
No, we haven't tested with local time stepping